### PR TITLE
Add missing copy of Except.h to copy list during pre-build

### LINKED ISSUE #311: CMake build fails for Windows
Steps to reproduce:

Clone, generate and build on a Windows box:

```cmd
git clone https://github.com/quickfix/quickfix.git
cd quickfix
mkdir build
cd build
cmake ..
cmake --build .
```

An excerpt from the output obtained:
```
...
2020-06-17T05:55:54.9282241Z   Building Custom Rule D:/a/quickfix/quickfix/examples/executor/C++/CMakeLists.txt
2020-06-17T05:55:54.9958658Z   Application.cpp
2020-06-17T05:55:56.5292286Z D:\a\quickfix\quickfix\include\quickfix\Utility.h(33,10): fatal error C1083: Cannot open include file: 'Except.h': No such file or directory [D:\a\quickfix\quickfix\build\examples\executor\C++\executor_cpp.vcxproj]
2020-06-17T05:55:56.7603364Z   executor.cpp
2020-06-17T05:55:57.0465468Z D:\a\quickfix\quickfix\include\quickfix\Utility.h(33,10): fatal error C1083: Cannot open include file: 'Except.h': No such file or directory [D:\a\quickfix\quickfix\build\examples\executor\C++\executor_cpp.vcxproj]
2020-06-17T05:55:57.0525035Z   Generating Code...
2020-06-17T05:55:57.1776089Z   Building Custom Rule D:/a/quickfix/quickfix/examples/ordermatch/CMakeLists.txt
2020-06-17T05:55:57.2657001Z   Application.cpp
2020-06-17T05:55:57.6641001Z D:\a\quickfix\quickfix\include\quickfix\Utility.h(33,10): fatal error C1083: Cannot open include file: 'Except.h': No such file or directory [D:\a\quickfix\quickfix\build\examples\ordermatch\ordermatch.vcxproj]
2020-06-17T05:55:57.6641758Z   Market.cpp
2020-06-17T05:55:58.2303009Z   ordermatch.cpp
2020-06-17T05:55:58.5252440Z D:\a\quickfix\quickfix\include\quickfix\Utility.h(33,10): fatal error C1083: Cannot open include file: 'Except.h': No such file or directory [D:\a\quickfix\quickfix\build\examples\ordermatch\ordermatch.vcxproj]
...
```

The above commands were executed as part of a github pipeline. The entire log output will be available for a while: https://github.com/tarc/quickfix/runs/779274019?check_suite_focus=true

### PR DESCRIPTION
**Pull request by tarc**
This fixes #311

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@ endif()
 
 if (WIN32)
 set (ut_SOURCES 
-${CMAKE_SOURCE_DIR}/src/C++/test/DataDictionaryTestCase.cpp
+# ${CMAKE_SOURCE_DIR}/src/C++/test/DataDictionaryTestCase.cpp
```
Comment (orenmnero) [inline src/CMakeLists.txt:25]: Why is this commented out?

Comment (tarc) [reply src/CMakeLists.txt:25]: I'm merging upstream master and will see if this is still necessary. I recall that I had to skip this test case in order for the pipeline to build.

In a few hours I'll return to you with the updated info.

Comment (tarc) [reply src/CMakeLists.txt:25]: It seems that the pipeline is catching some errors, I'm fixing the ones related to cmake here...

**Comment by orenmnero** [comment]
Except.h was already fixed in another PR, but saw the socket poll implementation which is great and solves a lot of problems for us. Made some adjustments for it to build everywhere and work with SSL. Brought in with #465

**Pull request closed without merging**